app/views/govuk_publishing_components/components/docs/chat_entry.yml

```diff
@@ -34,3 +34,6 @@ examples:
   with_margin_bottom:
     data:
       margin_bottom: 3
+  without_ga4_tracking:
+    data:
+      disable_ga4: true
```

Comment:
I think you should add the description to this like so https://github.com/alphagov/govuk_publishing_components/blob/7372183409497ce80237f0ed950bde9b697b3db2/app/views/govuk_publishing_components/components/docs/back_link.yml#L20

Reply:
Thanks for spotting that! Have updated